camel/societies/workforce/structured_output_handler.py

```
@@ -0,0 +1,500 @@
+                reasoning=f"Fallback decision due to: {error_message}",
+                modified_task_content=None,
+            )
+
+        else:
+            # Generic fallback
+            try:
+                return schema()
+            except Exception:
+                raise ValueError(
+                    f"Cannot create fallback for unknown schema: {schema_name}"
+                )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)